src/orchestration/Orchestration.ts

```diff
@@ -352,6 +353,17 @@ export class Orchestration {
         );
     }
 
+    /**
+     * Record an event directly to eventCache.
+     * @param type
+     * @param eventData
+     */
+    public recordEvent(eventType: string, eventData: object) {
+        if (!EVENT_TYPE_NAMESPACE.has(eventType)) {
```

Comment:
Removing this check allows users to send built-in events by creating their own data (assuming it conforms to the schema), which addresses a use case brought up before, and in the case where they do send wrong data format, at the end of the day, it will be handled by the Data Plane. Let me know what you think.

Reply:
Sounds good, as it also allows users to send built-in events on demand. Will reflect in next revision.